concourse/tasks/create_admin.yml

```diff
@@ -58,6 +58,7 @@ run:
         # N.B., It stopped printing `201 Created` during the upgrade to 
         # CF v269, and now just prints the `201` code on its own line
         # followed by a space.
+        # shellcheck disable=SC2181
```

Comment:
Agree that this is a good solution for now, but perhaps in future we could extract the `uaac curl` command so we can name the concept and also appease shellcheck.